docs/release_notes/release_notes-0.14.0.rst

```diff
@@ -23,6 +23,7 @@ Key Features and Updates
   *
 * Documentation improvements
   * DOCS-#4077: Add release notes template to docs folder (#4078)
+  * DOCS-#4082: Add pdf/epub/htmlzip formats for doc builds (#4083)
```

Comment:
@RehanSD I've added a type/issue number/PR name/ PR number as it's described in example above. But I didn't find an example where I should place my github username. Should it be placed in "Contributors" section or not? Example doesn't show where I should put it

Reply:
Yes please! For context, this is the release description you see [here](https://github.com/modin-project/modin/releases/tag/0.13.0), so please add your name to the Contributors section with an `@` so GitHub can also auto link your profile!

Reply:
Type/Issue/Pr Title + Number look perfect! I suspect Devin forgot to add his name to the contributors, which I'll fix when I add my docs fix PR that got merged last night to this template!